pandas/io/parquet.py

```diff
@@ -245,6 +245,17 @@ def to_parquet(
 
     kwargs
         Additional keyword arguments passed to the engine
+
+    Examples
+    --------
+    If you want get a buffer to the parquet content you can use a io.BytesIO object
+    >>> f = io.BytesIO()
+    >>> df.to_parquet(f)
+    >>> f.seek(0)
+    >>> len(f.read())
```

Comment:
maybe just assign `f.read()` to a variable for ease of use. What do you think?

Reply:
sure